internal/src/main/resources/application.yml

```
@@ -0,0 +1,42 @@
+    org:
+      hibernate:
+        orm:
+          jdbc:
+            bind: trace
+
+minio:
+  url: http://localhost:9000
+  access:
+    key: minio
+    secret: minioadmin
+  bucketName: original-bucket
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)